client/src/components/employer-form/OperatorInfo.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { FastField } from 'formik';
+
+import { Card, Divider } from '../generic';
+import { RenderTextField } from '../fields';
+
+export const OperatorInfo = ({ isDisabled }) => {
+
+  return (
+    <Card noShadow={isDisabled}>
+      <Grid container spacing={2}>
+
+        <Grid item xs={12}>
+          <Typography variant="subtitle2">
+            Operator Information
+          </Typography>
+          <Divider />
+        </Grid>
+
+        <Grid item xs={12}>
+          <FastField
+            name="operatorName"
+            component={RenderTextField}
+            label="* Operator Name:"
```

Comment:
This is a style difference I think we need to make a decision about as well - all of our forms use sentence case for labels, not title case